spark/dl/src/test/scala/com/intel/analytics/bigdl/nn/HighwaySpec.scala

```diff
@@ -42,7 +42,7 @@ class HighwaySpec extends KerasBaseSpec {
         |output_tensor = Highway(activation='tanh', bias=None)(input_tensor)
         |model = Model(input=input_tensor, output=output_tensor)
       """.stripMargin
-    val highway = Highway[Float](2, activation = "tanh", withBias = false)
+    val highway = Highway[Float](2, activation = Tanh[Float], withBias = false)
```

Comment:
why change this?

Reply:
Now using instance of activation on Scala side as other layers does.